solutions/pandas_tests/Makefile

```diff
@@ -0,0 +1,76 @@
+# Some tests in files listed in selective_tests fail/ error out on execution.
```

Comment:
Need to update solutions/Makefile to include pandas test in nightly pipeline.

Reply:
Instead of putting comments here, can you have a README file that documents them?